pkg/resource/hyper_parameter_tuning_job/testdata/v1alpha1/readone/observed/completed_variation.yaml

```diff
@@ -109,7 +109,6 @@ spec:
       volumeSizeInGB: 25
     roleARN: arn:aws:iam::123456789012:role/service-role/AmazonSageMaker-ExecutionRole-20210920T111639
     staticHyperParameters:
-      _tuning_objective_metric: validation:error
```

Comment:
revert?

Reply:
Isnt it not supposed to be there? https://github.com/aws-controllers-k8s/sagemaker-controller/blob/306948812591b0f32200ee6750a70fc0c4011649/pkg/resource/hyper_parameter_tuning_job/custom_delta.go#L25

Reply:
it will be there in observed returned from sdkFind. it is removed later on in the reconcile loop when delta is calculated

so for the unit test testing sdkfind, this is expected output